buildahimage/upstream/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# git/Dockerfile
+#
+# Build a Buildah container image from the latest
+# upstream version of Buildah on GitHub.
+# https://github.com/containers/buildah
+# This image can be used to create a secured container
+# that runs safely with privileges within the container.
+# The containers created by this image also come with a
+# Buildah development environment in /root/buildah.
+#
+FROM fedora
```

Comment:
I suggest pinning to a specific fedora version so that you know what you're getting.

Reply:
Would :latest work @TomasTomecek ?

Reply:
I think he wants :30 or something like that.  

But @TomasTomecek Since this is supposed to be latest upstream wouldn't we want the latest build chain tools.

Reply:
actually if latest is bumped to a "Fedora version + 1", you would probably be first to know that there is something wrong and can address right away

Reply:
Thanks @TomasTomecek .  I'm going to leave it as is with the ":latest" for all three and we'll roll the dice on the breakage not happening more frequently than me forgetting to bump the version number.